geth/jail/jail_rpc_test.go

```diff
@@ -42,27 +43,35 @@ func TestJailRPCTestSuite(t *testing.T) {
 
 type JailRPCTestSuite struct {
 	BaseTestSuite
-	Account common.AccountManager
-	Policy  *jail.ExecutionPolicy
+	Backend        *api.StatusBackend
+	TxQueueManager common.TxQueueManager
+	Account        common.AccountManager
+	Policy         *jail.ExecutionPolicy
 }
 
 func (s *JailRPCTestSuite) SetupTest() {
 	require := s.Require()
 
-	nodeman := node.NewNodeManager()
-	require.NotNil(nodeman)
+	backend := api.NewStatusBackend()
```

Comment:
You can make this test passing without getting `StatusBackend`. The only missing thing is to call 
```
txQueueManager.Start()
defer txQueueManager.Stop()
```

Getting the whole `StatusBackend` seems too heavy.

Reply:
Ah! I see, thank you.